src/components/ShellLine/index.tsx

```diff
@@ -20,48 +20,62 @@ export enum ShellLineType {
 export type ShellLineProps = {
   type: ShellLineType;
   message: string;
+  maxLength?: number;
   disabled?: boolean;
 };
 
-const ShellReadLine = ({ type, message, disabled }: ShellLineProps) => {
+const ShellReadLine = ({
+  type,
+  message,
+  disabled,
+  maxLength,
+}: ShellLineProps) => {
   const isPassword = message === 'password';
+
   return (
     <>
-      <span css={DefaultTypeStyle}>{type} </span>
-      <label htmlFor="readline" css={DefaultMessageStyle}>
-        {`${message}: `}
+      <span css={defaultTypeStyle}>{type}</span>
+      <label htmlFor="readline" css={readLineMessageStyle}>
+        {`${message}:`}
       </label>
-      <input
-        type={isPassword ? 'password' : 'text'}
-        css={InputStyle}
-        maxLength={MAX_SHELL_INPUT_LENGTH}
-        disabled={disabled}
-        autoComplete="off"
-        id="readline"
-      />
+      <span css={inputStyle}>
+        <input
+          type={isPassword ? 'password' : 'text'}
+          maxLength={maxLength}
+          disabled={disabled}
+          autoComplete="off"
+          id="readline"
```

Comment:
회원가입/로그인 페이지 리뷰하는 데 Shell 관련 컴포넌트 리뷰를 하게 되는 것 같아서 죄송하네요. 저번에 확인했어야 했는데....

요거 ShellLine 컴포넌트가 여러 개가 사용될 텐데 id 중복 문제는 없을까요?

Reply:
아닙니다!! 못짚고 넘어간 부분 짚어주시는 점은 항상 감사하죠 😄 
윤서님 말씀대로 id 중복 문제가 충분히 발생할 수 있다고 생각해, 
아래와 같이 `aria-labelledby` 속성을 사용하도록 수정하겠습니다 :)

```suggestion
          aria-labelledby="readline"
```

Reply:
명확하게 이런 상황에서 어떻게 해야 할지는 모르겠네요. 일단 종호님이 생각하신 대로 해봅시당!